user-interface/src/main/java/life/qbic/datamanager/views/general/contact/AutocompleteContactField.java

```diff
@@ -0,0 +1,178 @@
+package life.qbic.datamanager.views.general.contact;
+
+import static java.util.Objects.isNull;
+
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.customfield.CustomField;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.data.validator.EmailValidator;
+import java.util.ArrayList;
+import java.util.List;
+import life.qbic.datamanager.views.general.HasBinderValidation;
+
+/**
+ * <b>A component for contact person input</b>
+ *
+ * <p>Provides components to add a contact person with a name and email.
+ * Includes methods for basic input validation.</p>
+ *
+ * @since 1.0.0
+ */
+public class AutocompleteContactField extends CustomField<Contact> implements
+    HasBinderValidation<Contact> {
+
+
+  private final ComboBox<Contact> contactSelection;
+
+  private final ComboBox<String> nameField;
+  private final ComboBox<String> emailField;
```

Comment:
The decision to add a new combobox is different from the layout designed in the prototype. 
Was this discussed with Shraddha? 
An alternative could be to allow custom input in the name and email fields without a different selection combobox.

Reply:
? I do not understand this point.

Reply:
We have comboboxes here as they act as autocomplete fields. https://vaadin.com/docs/latest/components/combo-box#usage-as-autocomplete-field